README.md

```diff
@@ -89,7 +89,9 @@ is set to true, after successfull login the domain part will be striped and
 the rest used as username in Nextcloud. e.g. 'username@example.com' will be
 'username' in Nextcloud. The sixth parameter toggles whether on creation of
 the user, it is added to a group corresponding to the name of the domain part
-of the address. 
+of the address. The seventh parameter allows you specify - if is set - login
+options (for example, what authentication method you prefer).
```

Comment:
could you add the possible values for that seventh parameter to the documentation here ?

Reply:
Is this OK?